nototools/generate_website_2_data.py

```diff
@@ -435,38 +455,37 @@ def get_sample_infos(lang_scr):
     none: we have no attribution info on this, does not need attribution
   - source key.
   The list is in order of priority: language texts, udhr samples, exemplars for
-  the language, sample text for the script, exemplars for the script."""
+  the language, sample chars for the script, exemplars for the script."""
 
   assert '-' in lang_scr
 
   sample_infos = []
-  sample_text = get_sample_from_sample_file(lang_scr + '_text')
-  if sample_text is not None:
-    src_key = lang_scr + '_text'
-    attr = get_attribution(src_key)
-    sample_infos.append((sample_text, attr, src_key))
-
-  sample_text = get_sample_from_sample_file(lang_scr + '_udhr')
-  if sample_text is not None:
-    src_key = lang_scr + '_udhr'
-    attr = get_attribution(src_key)
-    sample_infos.append((sample_text, attr, src_key))
 
-  lang, script = lang_scr.split('-')
-  if lang != 'und':
+  def add_samples(lang_scr, typ):
+    for src_key in get_sample_names_for_lang_scr_typ(lang_scr, typ):
+      sample_text = get_sample_from_sample_file(src_key)
+      if sample_text is not None:
+        attr = get_attribution(src_key)
+        sample_infos.append((sample_text, attr, src_key))
+
+  def add_exemplars(lang_scr):
     exemplar, src_key = cldr_data.get_exemplar_and_source(lang_scr)
```

Comment:
Just a thought:  would it be better to include auxil. exemplar as well?  Well, for some languages, aux. exemplar chars have ZWJ/ZWNJ and can be messy so that it needs some filtering for display. Perhaps, not worth the trouble.

Reply:
Can you point me at a file with some?

Reply:
You mean ZWJ/ZWNJ?  For instance, Hindi has ZWJ/ZWNJ in AuxExemplarCharacters. (most other Indic languages do, too.  Bengali has a few more 'real' characters besides ZWJ/ZWNJ in AuxExemplar).

Reply:
common/main/hi.xml has this:
```
  <exemplarCharacters type="auxiliary">[\u200C\u200D]</exemplarCharacters>
```

Reply:
common/main/bn.xml has 
```
                <exemplarCharacters type="auxiliary">[\u200C\u200D ৲ ৳ ৴ ৵ ৶ ৷ ৸ ৹ ৰ ৱ]</exemplarCharacters>
```

Reply:
Not saying this has to be done now, but only including gC={L,M*,N} should work.